app/adapters/fetchallnews/adapter.rb

```diff
@@ -0,0 +1,13 @@
+module Fetchallnews
```

Comment:
i would suggest you change the module name to NewsFetcher(news_fetcher)
Fetchallnews is more appropriate for a method than a module